application/plugins/sms_member/views/js_member.php

```diff
@@ -22,16 +22,16 @@
 				$.post("<?php echo site_url('messages/compose_process') ?>", $("#composeForm").serialize(), function(data) {
 					$("#compose_sms_container").html(data);
 					$("#compose_sms_container" ).dialog( "option", "buttons", { "Okay": function() { $(this).dialog("destroy"); } } );
-					setTimeout(function() {$("#compose_sms_container").dialog('destroy')} , 1500);
+					setTimeout(function() {if ($("#compose_sms_container").hasClass('ui-dialog-content')) { $("#compose_sms_container").dialog('destroy')}} , 1500);
```

Comment:
Why this conditional check?

Reply:
Because if you click ok before the timeout, js will throw an error. So we need to check if it still exists before destroying it. Commit comment says:

`JQuery Error: cannot call methods on dialog prior to initialization;`